### DISCUSSION TITLE
is it adaptable for resturants and foods booking also?

### DISCUSSION THREAD

**Discussion by loveyandex** [vendurehq/vendure#1009] [Q&A]
is it adaptable for resturants and foods booking also? #1009

**Comment by michaelbromley** [comment]
Hi, yes you could probably adapt Vendure to work with this use-case.

**Reply by loveyandex** [reply]
how can i login as administrator and as customer?
is needed eveen customer login?
 can superadmin place an order for a customer with api?

**Reply by loveyandex** [reply]
@michaelbromley

**Reply by michaelbromley** [reply]
Hi,

It is not possible to log in as both a Customer and Administrator at the same time in the same browser session.

Administrators cannot place Order currently. I plan to add this functionality in a future release.

**Reply by loveyandex** [reply]
so you say i should use bearer token for every user?

**Reply by loveyandex** [reply]
thank you for your answers
